fluentlenium-core/src/test/resources/index.html

```diff
@@ -21,7 +21,7 @@
 <body style="" name="body">
 <h1>Heading</h1>
 
-<span id="oneline">A single line of text</span>
+<span id="oneline" onmouseover="javascript:document.getElementById('id').innerHTML='abc'">A single line of text</span>
```

Comment:
Maybe we coule keep this <span> unchanged, and add a new <span> for the purpose of this mouseover test.